istioctl/cmd/auth.go

```diff
@@ -19,18 +19,21 @@ import (
 	"io/ioutil"
 	"os"
 
-	"istio.io/istio/pkg/log"
-
 	"github.com/spf13/cobra"
+	k8s "k8s.io/client-go/kubernetes"
 
 	"istio.io/istio/istioctl/pkg/auth"
 	"istio.io/istio/istioctl/pkg/kubernetes"
 	"istio.io/istio/istioctl/pkg/util/configdump"
+	"istio.io/istio/pkg/kube"
+	"istio.io/istio/pkg/log"
 )
 
 var (
 	printAll       bool
 	configDumpFile string
+	v1PolicyFile   string
+	serviceSpecs   []string
```

Comment:
Sorry if I missed it, but does each serviceSpecs file contain the whole service config (name, namespace, etc.) or just the spec field? Looks like the second case based on the name?

Reply:
Changed to `serviceFiles`